src/AlgebraicGeometry/Surfaces/AdjunctionProcess/AdjunctionProcess.jl

```diff
@@ -199,22 +199,40 @@ SubquoModule{MPolyDecRingElem{fpFieldElem, fpMPolyRingElem}}
 ```
 """
 function canonical_bundle(X::AbsProjectiveVariety)
-   Pn = ambient_coordinate_ring(X)
-   n = ngens(Pn)-1
-   I = defining_ideal(X)
-   c = codim(I)
-   A, _ = quo(Pn, I)
-   FA = free_resolution(A, algorithm = :mres)
-   FAC = FA.C
-   r = range(FAC)
-   C = shift(FAC[first(r):-1:1], -c)
-   F = free_module(Pn, 1)
-   OmegaPn = grade(F, [n+1])
-   D = hom(C, OmegaPn);
-   Omega = homology(D, 0);
-   Omega, _ = prune_with_map(Omega);  
-   return Omega
+  Pn = ambient_coordinate_ring(X)
+  n = ngens(Pn)-1
+  I = defining_ideal(X)
+  c = codim(I)
+  A, _ = quo(Pn, I)
+  FA = free_resolution(A, algorithm = :fres)
+  C = Oscar.SimpleComplexWrapper(FA.C[0:first(range(FA.C))])
```

Comment:
```suggestion
  C = SimpleComplexWrapper(FA.C[0:first(range(FA.C))])
```

Just to be explicit: `SimpleComplexWrapper` is from `experimental`. Normally code in `src` should not depend on `experimental` code, but apparently this improvement is critical for a book example, plus hopefully `experimental/DoubleAndHyperComplexes` can migrate to `src` in the not-so distant future, so we could make an exception here.